### PR TITLE
Childhomepage

### PR DESCRIPTION
**Pull request by bluel1ner**

### CODE REVIEW

File: accounts/src/main/java/com/jungeeks/controller/ChildController.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import java.util.List;
+
+/**
+ * Rest controller for operations related to child home page:
+ * operation on getting deadline of all tasks
+ * operation getting data of all tasks
+ *
+ * @author nevels 09.29.2022
+ */
+@RequestMapping("/child/")
```
Comment (VolgaDaniluk) [inline accounts/src/main/java/com/jungeeks/controller/ChildController.java]: Use one style everywhere - “/path”

File: accounts/src/main/java/com/jungeeks/controller/ExController.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+@Slf4j
+public class ExController extends ResponseEntityExceptionHandler {
```
Comment (VolgaDaniluk) [inline accounts/src/main/java/com/jungeeks/controller/ExController.java]: Rename

File: accounts/src/main/java/com/jungeeks/response/TaskResponse.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import com.jungeeks.entity.enums.TASK_STATUS;
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Data
```
Comment (VolgaDaniluk) [inline accounts/src/main/java/com/jungeeks/response/TaskResponse.java:13]: What type of constructor do you need? There are 3 types now

File: accounts/src/main/java/com/jungeeks/service/entity/impl/UserServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+        return childs.getTasks().stream()
+                .map((child) ->
+                        TaskResponse.builder()
+                                .taskStatus(child.getTaskStatus())
+                                .title(child.getTask().getTitle())
+                                .point(child.getRewardPoints())
+                                .localDateTime(child.getDeadline())
+                                .build()
+                ).toList();
```
Comment (VolgaDaniluk) [inline accounts/src/main/java/com/jungeeks/service/entity/impl/UserServiceImpl.java]: Move “)” to the previous row

File: persistence/src/main/java/com/jungeeks/entity/ChildNotification.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Map;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Data
```
Comment (VolgaDaniluk) [inline persistence/src/main/java/com/jungeeks/entity/ChildNotification.java]: Same Q about constructors

File: persistence/src/main/java/com/jungeeks/entity/ParentNotification.java [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Map;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Data
```
Comment (VolgaDaniluk) [inline persistence/src/main/java/com/jungeeks/entity/ParentNotification.java]: Same Q about constructors

File: accounts/src/main/java/com/jungeeks/controller/ChildController.java (hunk 1 above)
Comment (bluel1ner) [reply accounts/src/main/java/com/jungeeks/controller/ChildController.java]: fix

File: accounts/src/main/java/com/jungeeks/controller/ExController.java (hunk 2 above)
Comment (bluel1ner) [reply accounts/src/main/java/com/jungeeks/controller/ExController.java]: fix

File: accounts/src/main/java/com/jungeeks/service/entity/impl/UserServiceImpl.java (hunk 4 above)
Comment (bluel1ner) [reply accounts/src/main/java/com/jungeeks/service/entity/impl/UserServiceImpl.java]: fix

File: persistence/src/main/java/com/jungeeks/entity/ParentNotification.java [hunk 7]
```diff
@@ -0,0 +1,33 @@
+package com.jungeeks.entity;
+
+import com.jungeeks.entity.enums.NOTIFICATION_PERIOD;
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.Map;
+@AllArgsConstructor
```
Comment (VolgaDaniluk) [inline persistence/src/main/java/com/jungeeks/entity/ParentNotification.java:8]: line between imports and annotations

**Review by VolgaDaniluk — APPROVED** [review]

**Pull request merged**